services/meeting_service.js

```diff
@@ -23,7 +25,7 @@ const areNewHosts = async (meeting,hostsMails) => {
     return JSON.stringify(meetingMails) === JSON.stringify(hostsMails)
```

Comment:
Wiem że nie z tego pr, ale wgl tu błąd "emial"
I tak się zastanawiam, czy mamy zagwarantowane, że hostsMails i meetingMails będą miały identyczną kolejność maili?

Reply:
W sumie nie mamy ale to w takim razie sortowanie powinno załatwić sprawę, ale gratki za czujność